<launch>

    <node pkg="object_detection_sensor_fusion" type="object_detector_3d" name="object_detector_3d">
        <remap from="/camera/color/image_raw" to="/camera0/color/image_raw"/>
        <remap from="/camera/color/camera_info" to="/camera0/color/camera_info"/>
        <remap from="/velodyne_points" to="/rm_ground"/>
        <remap from="/projection" to="/projection"/>
        <remap from="/colored_cloud" to="/colored_cloud"/>
        <remap from="/cloud/colored/semantic" to="/cloud/colored/semantic"/>
        <remap from="/darknet_ros/bounding_boxes" to="/darknet_ros/bounding_boxes"/>
        <param name="LEAF_SIZE" value="0.05"/>
        <param name="TOLELANCE" value="0.20"/>
        <param name="MIN_CLUSTER_SIZE" value="2"/>
        <param name="MAX_CLUSTER_SIZE" value="900"/>
    </node>
</launch>
